bladex/profilebase.py

```diff
@@ -202,6 +202,62 @@ def compute_camber_line(self, interpolate=False, n_interpolated_points=500):
             self.camber_line = np.array(
                                [self.xup_coordinates, cl_y_coordinates])
 
+    def deform_camber_line(self, max_camber_change_percent=None,
```

Comment:
Instead of checking whether `max_camber_change_percent` is None or not, do not assign a default value. Moreover you can rename the argument to `percent_change`